extra/3-magic-8-ball.js

```diff
@@ -43,10 +43,50 @@
     Very doubtful.
 */
 
+// This should log "The ball has shaken!"
+// and return the answer.
+const veryPositive = [
+  "It is certain.",
+  "It is decidedly so.",
+  "Without a doubt.",
+  "Yes - definitely.",
+  "You may rely on it.",
+];
+
+const positive = [
+  "As I see it, yes.",
+  "Most likely.",
+  "Outlook good.",
+  "Yes.",
+  "Signs point to yes.",
+];
+
+const negative = [
+  "Reply hazy, try again.",
+  "Ask again later.",
+  "Better not tell you now.",
+  "Cannot predict now.",
+  "Concentrate and ask again.",
+];
+
+const veryNegative = [
+  "Don't count on it.",
+  "My reply is no.",
+  "My sources say no.",
+  "Outlook not so good.",
+  "Very doubtful.",
+];
+
 // This should log "The ball has shaken!"
 // and return the answer.
 function shakeBall() {
-  //Write your code in here
+  const allAnswers = [
+    ...veryPositive,
+    ...positive,
+    ...negative,
+    ...veryNegative,
+  ];
+  return allAnswers[Math.round(Math.random() * allAnswers.length)];
```

Comment:
Thank you so much for your resubmission and I should have found out it much earlier! Nice and quality work as before. Refer to line 80, the function should log "The ball has shaken" to console when it is called